packages/core/src/icon/utils/icon.classnames.spec.ts

```diff
@@ -99,7 +99,7 @@ describe('Icon classname helpers: ', () => {
     let component: CdsIcon;
 
     beforeAll(() => {
-      ClarityIcons.add({ testing: testIcon });
+      ClarityIcons.addIcons(['testing', testIcon]);
```

Comment:
There's a lot of code in this spec and corresponding utility that is no longer needed. Please remove.